bin/moodle-compose

```diff
@@ -92,7 +93,36 @@ if [ -n "$1" ]; then
         ARGS="exec webserver php admin/cli/cron.php"
         shift 1
         ;;
+    pgrestore|pgr)
+        if [ -z $2  ]; then
+	        echo "Insert the name of the pgdump file: "
+	        read dump
+        else
+	        dump=$2
+        fi
+        clear
+        printf "\n>>> Copying the pgdump file to the ${MOODLE_CODENAME}_db_1  volume\n------------------------------------------------------\n"
+        # It depends on Python, but who does not have it this days?
+        destination_folder=$(docker inspect -f '{{ json .Mounts }}' ${MOODLE_CODENAME}_db_1 | python -m json.tool | grep Destination | cut -d':' -f2 | cut -d'"' -f2)
+        date=$(date +%Y%m%d)
+        filename=${destination_folder}/${MOODLE_CODENAME}.${date}.${dump}
+        printf "${filename}\n"
+        docker cp ${dump} ${MOODLE_CODENAME}_db_1:${filename}
+        printf "\n>>> Removing the moodle DB\n--------------------------\n"
+        docker exec -t ${MOODLE_CODENAME}_db_1 /usr/bin/dropdb -e -U moodle moodle
```

Comment:
I'd replace all the `docker exec -t ${MOODLE_CODENAME}_db_1` with `moodle-compose exec db`; that's much more robust.

Reply:
Solved in another merge request, this one can be dismissed.